### DISCUSSION TITLE
Strange training issue.. I think..

### DISCUSSION THREAD

**Discussion by brunopiras** [skrashevich/double-take#143] [General]
Hello everyone,

I'm noticing a strange behavior when training double-take + deepstack, on the "Matches" tab when one of my images to the camera detects me as unknown, I select it and then at the top I choose my name and press "Train" ok on that image but it's like it's erasing the work done with previous images.

Am I doing something wrong? Is it correct for it to behave this way? 
Thank you very much

**Comment by skrashevich** [comment]
It's really strange. Please, create new issue in this repo for this problem with complete step-to-reproduce
